### PR TITLE
fix(ui5-SegmentedButton): keyboard handling now compliant with the specification

### PR DESCRIPTION
**Pull request by Todor-ads**
Focus is moved on the first element of Segmented Button or last element via PageUp or PageDown keyboard key.
Related to: https://github.com/SAP/ui5-webcomponents/issues/3091

### CODE REVIEW

File: packages/main/src/SegmentedButton.js [hunk 1]
```diff
@@ -213,7 +218,25 @@ class SegmentedButton extends UI5Element {
...
+		} else if (isPageUp(event)) {
+			event.preventDefault();
+			this._onPageUpOrDown(isPageUp(event));
+		} else if (isPageDown(event)) {
+			event.preventDefault();
+			this._onPageUpOrDown();
+		}
+	}
+
+	_onPageUpOrDown(pageUpPressed) {
```
Comment (unazko) [inline packages/main/src/SegmentedButton.js:230]: This function doesn't make the code more readable or structured. Better execute the code on place.

File: packages/main/src/SegmentedButton.js [hunk 2]
```diff
@@ -213,7 +218,25 @@ class SegmentedButton extends UI5Element {
 			this._selectItem(event);
 		} else if (isSpace(event)) {
 			event.preventDefault();
+		} else if (isPageUp(event)) {
+			event.preventDefault();
+			this._onPageUpOrDown(isPageUp(event));
```
Comment (ilhan007) [inline packages/main/src/SegmentedButton.js:223]: if you are already in this "if" block I think isPageUp(event) in "this._onPageUpOrDown(isPageUp(event))" is always true as already checked "else if (isPageUp(event)"

File: packages/main/src/SegmentedButton.js (hunk 1 above)
Comment (Todor-ads) [reply packages/main/src/SegmentedButton.js]: Done

File: packages/main/src/SegmentedButton.js (hunk 2 above)
Comment (Todor-ads) [reply packages/main/src/SegmentedButton.js]: Done

**Review by unazko — COMMENTED** [review]
The code now seems good to me.

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**